storage/plugins/resource-sync/ResourceSyncPlugin.php

```
@@ -0,0 +1,459 @@
+        } else {
+            $stmt = $this->db->prepare(
+                'SELECT id, updated_at, created_at, deleted_at, 0 AS is_new_entry
+                 FROM libri
+                 WHERE deleted_at IS NULL
+                 ORDER BY updated_at DESC
+                 LIMIT 500'
+            );
+            if ($stmt === false) {
+                return [];
+            }
+        }
```

PHPMD (2.15.0)

[error] The method fetchChangedBooks uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)